tsconfig.json

```diff
@@ -1,14 +1,23 @@
 {
-  "extends": "../zlux-app-manager/virtual-desktop/plugin-config/tsconfig.base.json",
```

Comment:
Why does this no longer extend?

Reply:
The commit that had "extends" was broken, so to move forward, I just went with what was working.